### PR TITLE
Migrate from electron-builder to electron-forge

### PR DESCRIPTION
**Pull request by SFR-git**
[`electron-forge`](https://electronforge.io) supports Flatpak packaging, so we're moving from electron-builder to it.

**Global**:
- [x] Get stylesheets to work
- [x] License
- [x] `appId`
- [x] `productName`
- [x] Update README

**Flatpak**:
- - [x] Fix zypak bug
- - [ ] Display icon

**DMG**:
- - [x] Display correct icons 
- - [x] Icon placements

**Squirrel.Windows**:
- - [x] Display icon

### CODE REVIEW

**Review by SFR-git — COMMENTED** [review]
This will allow me to use `require('path')` and be able to change the macOS icon placements without messing up their paths.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,5 @@
 {
     "name": "stonecutter-electron",
-    "productName": "Stonecutter",
```
Comment (SFR-git) [inline package.json]: This *will* mess up macOS' titlebar name, so we'll need to find a workaround

Comment (SFR-git) [reply package.json]: Fixed by 5d93198.

**Review by SFR-git — COMMENTED** [review]
This succesfully builds and installs on Flatpak;
@3174N > Please check that 5d93198 shows the correct title on Windows.

**Comment by 3174N** [comment]
Everything works as intended!

image
image

**Comment by SFR-git** [comment]
Since the Flatpak icon problem doesn't seem to have a fix, I'm marking this PR as ready for review, and if we can't find a solution to it within the next couple of days, it will be merged.

**Comment by SFR-git** [comment]
Tracking of the icon problem has been moved to #23, and this PR will be merged without it.

**Pull request merged**